nsv13/game/custom_turfs.dm

```diff
@@ -36,7 +60,31 @@
 			transfer_fingerprints_to(T)
 			qdel(src)
 			return
-	. = ..()
+
+	else //Build false wall
+		if(S.get_amount() < 2)
+			to_chat(user, "<span class='warning'>You need two sheets of [S] to create a false wall!</span>")
+			return
+
+		to_chat(user, "<span class='notice'>You start building a false wall with [S]...</span>")
+		if (do_after(user, 40, target = src))
+			if(S.get_amount() < 2)
+				return
+			S.use(2)
+			to_chat(user, "<span class='notice'>You create a false wall. Push on it to open or close the passage.</span>")
+
+			var/turf/T = get_turf(src)
+			var/obj/structure/falsewall/FW
+			if (istype(S, /obj/item/stack/sheet/durasteel))
```

Comment:
Why are these checks, `turf_type` contains the required path.

Reply:
Only for real walls, not false ones